yt_dlp/downloader/hls.py

```diff
@@ -262,12 +263,20 @@ def is_ad_fragment_end(s):
             return fd.real_download(filename, info_dict)
 
         if is_webvtt:
+            lenient = params.get('webvtt_lenient', False)
+            if lenient:
+                self.report_warning(bug_reports_message(
+                    'The extractor has signalled that this site may serve '
+                    'corrupt subtitles; a lenient parser that should be able '
+                    'to correct this has been activated. '
+                    'If the downloaded subtitles are still wrong,'))
```

Comment:
I don't think this is warranted. Either, subtitles will break and user will report it anyway, or there is nothing for user to do about this warning. Perhaps a debug message can be added instead to help developers

Reply:
My intent here is that the warning serve two purposes:
- Making the user aware that they may need to inspect the download and should not just assume everything is fine, only to find out the subtitles are corrupt weeks later, after the stream link expires.
- Shaming the website about serving broken subtitles; this is the least we could do about it. [Sometimes shaming works.](https://arstechnica.com/information-technology/2017/03/firefox-gets-complaint-for-labeling-unencrypted-login-page-insecure/). And I think the serving site does have an interest in fixing this issue, as it may break even their intended web player.